machine/inventec/inventec_d7054q28b/machine.make

```diff
@@ -0,0 +1,55 @@
+# Inventec D7054Q28B
+# CPU Module: Intel Broadwell_DE (C2000)
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = bcm
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Inventec Corporation
+VENDOR_ID = 6569
+
+# Skip the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
```

Comment:
It is no longer necessary to specify `I2CTOOLS_SYSEEPROM = no`, as that is the default.